kuwala/core/canvas/src/components/Modals/TransformationCatalog/TransformationCatalogModal.js

```diff
@@ -0,0 +1,329 @@
+import React, {useEffect, useState} from "react";
+import {useStoreActions} from "easy-peasy";
+import {
+    getAllTransformationCatalogCategories,
+    getAllItemsInCategory
+} from "../../../api/TransformationCatalog";
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {faShuffle} from "@fortawesome/free-solid-svg-icons";
+import {getCatalogItemIcon} from "../../../utils/TransformationCatalogUtils";
+import {getDataDictionary} from "../../../utils/SchemaUtils";
+import ReactTable from "react-table-6";
+import "./transformation-example-table.css";
+import Modal, {ModalHeader, ModalFooter, ModalBody} from "../../Common/Modal";
+import Button from "../../Common/Button";
+import CloseButton from "../../Common/CloseButton";
+import Classes from "./TransformationCatalogStyle";
+import cn from "classnames";
+
+const ExampleTable = ({columns, rows}) => {
+    let populatedColumns = columns.map((el,i)=>{
+        return {
+            Header: el,
+            accessor: el,
+        }
+    });
+
+    populatedColumns = [{
+        Header: "",
+        id: "row",
+        filterable: false,
+        width: 50,
+        Cell: (row) => {
+            return <div>{row.index+1}</div>;
+        }
+    }, ...populatedColumns]
+
+    const populatedRows = getDataDictionary(rows, columns);
+
+    return (
+        <div className={'transformation-example h-full'}>
+            <ReactTable
+                data={populatedRows}
+                columns={populatedColumns}
+                defaultPageSize={3}
```

Comment:
Three rows are too low as some examples contain four atm. So I'd set it to 10 to make sure it's enough also in the future.

Reply:
Resolved set the max default page to 10 when data >= 10 and will follow it's own length when data <= 10